pyron-engine/src/main/scala/com/cloudentity/pyron/api/ApiHandler.scala

```diff
@@ -82,47 +84,59 @@ class ApiHandlerVerticle extends ScalaServiceVerticle with ApiHandler with ApiGr
     val vertxRequest   = ctx.request()
     val vertxResponse  = ctx.response()
     val tracingContext = RoutingWithTracingS.getOrCreate(ctx, getTracing)
+    val proxyHeaders   = getProxyHeaders(ctx)
 
     val requestSignature = Option(vertxRequest.host()) match {
       case Some(_) => s"${vertxRequest.method()} ${vertxRequest.uri()}"
       case None => s"${vertxRequest.method()} ${vertxRequest.absoluteURI()}"
     }
-    log.debug(tracingContext, s"Received request: $requestSignature")
```

Comment:
Why get rid of this log line?

Reply:
It's redudant as we have that info in the access log